.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ workflows:
 jobs:
   build-ubuntu-image:
     environment:
-      IMAGE_NAME: haugene/transmission-openvpn
+      IMAGE_NAME: steamgerbil/transmission-openvpn
```

Comment:
This is not right, please rebase and squash your commits and remove these circle config as they are pointing at the wrong image

Reply:
Sorry I thought I was on a cloned repo and not the fork, I've reverted those changes so only config changes are in. I'll rebase and push